diagnostics-app/src/app/core/services/telemetry.service.ts

```diff
@@ -106,4 +107,10 @@ export class TelemetryService implements TelemetryInterface {
   fetchVpdInfo() {
     return <Promise<VpdInfo>>this.fetchTelemetryData(TelemetryInfoType.VPD);
   }
+
```

Comment:
The <code>fetchInfo</code> function is removed. And the <code>fetchTelemetryData</code> function is changed to be public so that UI can call it to get the requested Telemetry data.